docs/product-user-guide/src/main/asciidoc/guided-rule-templates-tables-proc.adoc

```diff
@@ -24,7 +24,9 @@ Example:
 rule "PaymentRules_6"
 	dialect "mvel"
 	when
-		Customer( internetService == false , phoneService == false , TVService == true )
+		Customer( internetService == false ,
+			phoneService == false ,
+			TVService == true )
```

Comment:
Sorry, probably I did mistake in my original PR, letters **i**nternetService, **p**honeService and **T**VService should have the same indentation.

Reply:
Thanks for an explanation